src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorMenuItem.class.php

```diff
@@ -208,14 +217,26 @@ protected function renderItemWithSubMenu()
             $aSubItemData['sItemKey'] = $oSubItem->sItemKey;
             $aSubItemData['sCSSClass'] = $oSubItem->sCSSClass;
             $aSubItemData['sOnClick'] = $oSubItem->sOnClick;
-            $aSubItemData['sIconURL'] = $oSubItem->sIcon;
+            if ($this->iconIsURL($oSubItem->sIcon)) {
+                $aSubItemData['sIconURL'] = $oSubItem->sIcon;
+            } else {
+                $aSubItemData['sIcon'] = $oSubItem->sIcon;
+            }
             $aSubItems[] = $aSubItemData;
         }
         $oViewRenderer->AddSourceObject('aSubItems', $aSubItems);
 
         return $oViewRenderer->Render('MTTableEditor/menuButtonWithDropdown.html.twig', null, false);
     }
 
+    private function iconIsURL($icon) {
```

Comment:
--> `isIconUrl` (begin method with `is`, camel-case)

+ line break before the bracket is missing
+ return type `bool` is missing

Reply:
done